src/uct/tcp/tcp.h

```diff
@@ -16,6 +16,39 @@
 /** How many events to wait for in epoll_wait */
 #define UCT_TCP_MAX_EVENTS        32
 
+#define UCT_TCP_AM_SHORT_PACK_DATA(_pack_f, _target_buf, _target_length, \
+                                   _am_payload, _payload_length, _am_header) \
+    do { \
+        _pack_f(_target_buf, &(_am_header), sizeof(_am_header)); \
```

Comment:
could just assign header in `uct_tcp_ep_am_short` (instead of extra memcpy) and have one general macro instead of 3.

Reply:
done